Assets/Global/Scripts/Platform/PlatformPathController.cs

```diff
@@ -0,0 +1,121 @@
+using UnityEngine;
+using System.Collections.Generic;
+using System.Collections;
+
+public class PlatformPathController : MonoBehaviour
+{
+    public Transform pathParent; 
+    private List<Vector3> controlPoints = new List<Vector3>();
+    public float platformSpeed = 5f; 
+    public float tStartPause = 0f; 
+    public float tEndPause = 0f;
+    private bool pausedTime = false;
+    public bool pause = false;
+    private float pauseTimer = 0f;
+
+    private float t = 0f; 
+    private int currentSegment = 0;
+    private bool isReversing = false; 
+
+    private void Start()
+    {
+        foreach (Transform child in pathParent)
+        {
+            controlPoints.Add(child.position);
+        }
+    }
+
+
+private void FixedUpdate()
+{
+    if (pause) return; 
```

Comment:
agreed, but i think she put it there so that you can manually pause it using the insepctor :)
not sure thought but it probably is because its public and not private